database/seeds/AvailabilitySeeder.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+use Illuminate\Database\Seeder;
+
+class AvailabilitySeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        factory(\App\Modules\Availability\Availability::class, 20)->create();
```

Comment:
Don't need that many, as likely going to create our own in tests
```suggestion
        factory(\App\Modules\Availability\Availability::class, 5)->create();
```

Reply:
done